### PR TITLE
one more set of patterns

### PR DESCRIPTION
**Pull request by gearama**
# Pull Request Checklist

Please leverage this checklist as a reminder to address commonly occurring feedback when submitting a pull request to make sure your PR can be reviewed quickly:

See the detailed list in the [contributing guide](https://github.com/Azure/azure-sdk-for-cpp/blob/main/CONTRIBUTING.md#pull-requests).

- [ ] [C++ Guidelines](https://azure.github.io/azure-sdk/cpp_introduction.html)
- [ ] Doxygen docs
- [ ] Unit tests
- [ ] No unwanted commits/changes
- [ ] Descriptive title/description
  - [ ] PR is single purpose
  - [ ] Related issue listed
- [ ] Comments in source
- [ ] No typos
- [ ] Update changelog
- [ ] Not work-in-progress
- [ ] External references or docs updated
- [ ] Self review of PR done
- [ ] Any breaking changes?

### CODE REVIEW

**Review by LarryOsterman — APPROVED** [review]

File: sdk/core/azure-core-test/src/test_proxy_manager.cpp [hunk 1]
```diff
@@ -243,6 +243,8 @@ void TestProxyManager::SetProxySanitizer()
   addSanitizer(SanitizerType::General, g_accountRegex, "account");
   addSanitizer(SanitizerType::Body, "client_secret=(?<clientsecret>[^&]+)", "clientsecret");
   addSanitizer(SanitizerType::Body, "client_id=(?<clientid>[^&]+)", "clientid");
+  addSanitizer(SanitizerType::Body, "(?:<Value>)(.*)(?:</Value>)", "value");
```
Comment (ahsonkhan) [inline sdk/core/azure-core-test/src/test_proxy_manager.cpp:246]: What's the `Value` here, typically, and is it sensitive info? This one seems pretty generic to redact.

File: sdk/storage/README.md [hunk 2]
```diff
@@ -36,3 +36,4 @@ additional questions or comments.
 [coc]: https://opensource.microsoft.com/codeofconduct/
 [coc_faq]: https://opensource.microsoft.com/codeofconduct/faq/
 [coc_contact]: mailto:opencode@microsoft.com
+
```
Comment (ahsonkhan) [inline sdk/storage/README.md:39]: Remove unrelated change

**Review by RickWinter — APPROVED** [review]

File: sdk/storage/README.md (hunk 2 above)
Comment (gearama) [reply sdk/storage/README.md:39]: i need it to trigger the storage pipeline

File: sdk/core/azure-core-test/src/test_proxy_manager.cpp (hunk 1 above)
Comment (gearama) [reply sdk/core/azure-core-test/src/test_proxy_manager.cpp:246]: it's actually an xml encoded thng the regex says look for Value.../Value

Comment (gearama) [reply sdk/core/azure-core-test/src/test_proxy_manager.cpp:246]: show only in a handful of storage recordings

Comment (LarryOsterman) [reply sdk/core/azure-core-test/src/test_proxy_manager.cpp:246]: FWIW, that should only affect SDKs whose bodies are encoded as XML, which I *think* only affects Storage.

Comment (ahsonkhan) [reply sdk/core/azure-core-test/src/test_proxy_manager.cpp:246]: From @alzimmermsft
> Okay, they do use \<Value>x\</Value> elsewhere when the value isn't a secret in Blob Tags, that regex will need a look around for \<UserDelegationKey\>

Comment (gearama) [reply sdk/core/azure-core-test/src/test_proxy_manager.cpp:246]: digging a bit deeper into this value in userdelegationkey in the case of those combos the value is already redacted 
E.G. \u003CValue\u003E**REDACTED**\u003C/Value\u003E\u003C/UserDelegationKey

**Comment by gearama** [comment]
digging a bit deeper into this value into user delegation, we already redact it 
E.G. \u003C**Value**\u003E**REDACTED**\u003C/**Value**\u003E\u003C/UserDelegationKey

File: sdk/core/azure-core-test/src/test_proxy_manager.cpp [hunk 3]
```diff
@@ -243,6 +243,7 @@ void TestProxyManager::SetProxySanitizer()
   addSanitizer(SanitizerType::General, g_accountRegex, "account");
   addSanitizer(SanitizerType::Body, "client_secret=(?<clientsecret>[^&]+)", "clientsecret");
   addSanitizer(SanitizerType::Body, "client_id=(?<clientid>[^&]+)", "clientid");
+  addSanitizer(SanitizerType::Body, "(?:<SignedTid>)(.*)(?:</SignedTid>)", "signedtid");
```
Comment (ahsonkhan) [inline sdk/core/azure-core-test/src/test_proxy_manager.cpp:246]: The regex pattern shared by the Java team is now:
```text
(?<=<UserDelegationKey>).*?(?:<SignedTid>)(.*)(?:</SignedTid>)
```

**Review by ahsonkhan — APPROVED** [review]

File: sdk/core/azure-core-test/src/test_proxy_manager.cpp (hunk 3 above)
Comment (gearama) [reply sdk/core/azure-core-test/src/test_proxy_manager.cpp]: just updated and also added Signed oid which just popped up

**Comment by gearama** [comment]
@Jinming-Hu a couple more regexes for test proxy that will require test re-recording, apologies for the randomization, they showed up a couple of hours ago and i implemented them. Hopefully this will be over soon and we will have a centralized repository of regexes in the test-proxy
